polygeist/test/polygeist-opt/loop-restructure-keep-block-arg.mlir

```diff
@@ -0,0 +1,59 @@
+// RUN: polygeist-opt --allow-unregistered-dialect --loop-restructure %s | FileCheck %s
+
```

Comment:
Maybe we can add an example of C code that would generate this input here.

Reply:
Done